### ISSUE TITLE
[React] It does not work properly in webpack 5.24.4 version.

### ISSUE THREAD

**Issue by dragonsuperf** [nhn/tui.chart#619]
### Describe the bug

The following error occurs at runtime when running in webpack 5.24.4 version.
![image](https://user-images.githubusercontent.com/23359043/110943863-0d2bed80-837f-11eb-9fde-cc496a9e5c83.png)

I will attach the contents of package.json so that you can reproduce it.

```
  "dependencies": {
    "@babel/core": "^7.13.10",
    "@babel/preset-env": "^7.13.10",
    "@babel/preset-react": "^7.12.13",
    "@babel/preset-typescript": "^7.13.0",
    "@testing-library/jest-dom": "^5.11.9",
    "@testing-library/react": "^11.2.5",
    "@toast-ui/react-chart": "^4.1.4",
    "@types/react": "^17.0.3",
    "@types/react-chartjs-2": "^2.5.7",
    "@types/react-dom": "^17.0.2",
    "babel-jest": "^26.6.3",
    "babel-loader": "^8.2.2",
    "css-loader": "^5.1.2",
    "error-overlay-webpack-plugin": "^0.4.2",
    "html-loader": "^2.1.2",
    "html-webpack-plugin": "^5.3.1",
    "mini-css-extract-plugin": "^1.3.9",
    "prettier": "^2.2.1",
    "react": "^17.0.1",
    "react-chartjs-2": "^2.11.1",
    "react-dom": "^17.0.1",
    "source-map-loader": "^2.0.1",
    "ts-loader": "^8.0.17",
    "ts-node": "^9.1.1",
    "typescript": "^4.2.3",
    "webpack": "^5.24.4",
    "webpack-cli": "^4.5.0"
  },
  "devDependencies": {
    "jest": "^26.6.3",
    "jest-config": "^26.6.3",
    "@jest/types": "^26.6.2",
    "@typescript-eslint/eslint-plugin": "^4.17.0",
    "@typescript-eslint/parser": "^4.17.0",
    "eslint": "^7.21.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-jsx-a11y": "^6.4.1",
    "eslint-plugin-prettier": "^3.3.1",
    "eslint-plugin-react": "^7.22.0",
    "eslint-plugin-react-hooks": "^4.2.0",
    "webpack-dev-server": "^3.11.2"
  }
```

### Additional context
After downgrading the webpack version to 4.44.2, it works normally

**Comment by jung-han** [comment]
@dragonsuperf 

I think we need to test more in webpack5 in react chart. Thanks for rasing issue! 
I'll let you know when it is fixed.

**Comment by detygon** [comment]
I'm facing the same issue with the vue adapter when running webpack 5.25.
![tui-error](https://user-images.githubusercontent.com/25983025/111890431-236e2500-89e1-11eb-8689-0a901c4d5eca.png)

Here are my dependencies:

```
  "devDependencies": {
    "laravel-mix": "^6.0.13",
    "vue-loader": "^15.9.6",
    "vue-template-compiler": "^2.6.12"
  },
  "dependencies": {
    "@toast-ui/vue-chart": "^4.2.0",
    "vue": "^2.6.12"
  }
```

**Comment by jung-han** [comment]
@dragonsuperf @detygon 

The problem will be fixed and released in version 4.2.1. The wrapper didn't provide an ESM, so the problem could be fixed. However, it was not possible to understand exactly how it was affected by the user webpack version, so the issue was posted on the webpack repo. I'll let you know when that version is available. Thank you!

**Comment by jung-han** [comment]
@dragonsuperf @detygon 
Try it!
https://github.com/nhn/tui.chart/releases/tag/v4.2.1
